html/global_attributes.json

```diff
@@ -601,6 +606,7 @@
         "__compat": {
           "description": "<code>data-*</code> attributes",
           "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Global_attributes/data-*",
+          "spec_url": "https://html.spec.whatwg.org/multipage/dom.html#embedding-custom-non-visible-data-with-the-data-*-attributes",
```

Comment:
#attr-data-* would be an alternative. I think we need to make a call if we generally want the headline or the #attr-xyz link.

Reply:
> #attr-data-* would be an alternative

Good point — changed to that in 822f492 for consistency

>  I think we need to make a call if we generally want the headline or the #attr-xyz link.

It seems like we lean toward wanting to use the most-specific anchors, and the changes made from this review have tended to do that; so perhaps for consistency we should try to continue to follow that policy.

Reply:
Yeah, I think that's a good policy.